src/core/ext/xds/xds_api.cc

```diff
@@ -57,7 +57,7 @@
 namespace grpc_core {
 
 XdsApi::XdsApi(XdsClient* client, TraceFlag* tracer,
-               const XdsBootstrap::Node* node, upb::SymbolTable* symtab,
+               const XdsBootstrap::Node* node, upb::DefPool* symtab,
```

Comment:
Please also change the name of the variables everywhere from "symtab" to "def_pool".